slices/util-linux.yaml

```
@@ -0,0 +1,135 @@
+package: util-linux
+
+essential:
+  - util-linux_copyright
+
+slices:
+  bins:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libcrypt1_libs
+      - libmount1_libs
+      - libpam0g_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libsystemd0_libs
+      - libtinfo6_libs
+      - libudev1_libs
+      - libuuid1_libs
+      - util-linux_generated
```

```suggestion
      - util-linux_generated
      - util-linux_pam-profile
```
I believe `su` requires those files in pam-profile slice to run.

---

Affirm, `su` does need those files.